### DISCUSSION TITLE
Eigenschaften für Produktlisting und Detailseite

### DISCUSSION THREAD

**Discussion by ToniMarokoko** [affiliatetheme/affiliatetheme#5] [Q&A]
Peace, ich benutze das Guide Plugin und habe hier eine Template erstellt mit den Affiliate Theme Produktboxen.

Gerne möchte ich jetzt auf der Detailseite Eigene Felder ausgeben, die sich von denen in den Produktboxen unterscheiden.

..In den Produktboxen möchte ich nur 4-5 Eigenschaften anzeigen, auf den Produktdetailseiten ein paar mehr. 

Beispiel-Screenshots:

![Detailseite](https://user-images.githubusercontent.com/26651966/216303749-a6da6e7e-26c7-4280-b638-606f39feb9f6.png)
![Produktbox](https://user-images.githubusercontent.com/26651966/216303756-a2f708f6-f361-4079-a541-09c720b83da3.png)

Freue mich über eine Lösung

**Comment by olaf-schmitz** [comment]
Das müsstest Du dann im Child Theme selber in die Funktion der Produktbox einfügen. Kannst es dann gerne bereitstellen und wir prüfen es und binden es dann ggf. in den Code ein.
